src/dataprep/main/link/setup_linking.py

```diff
@@ -144,6 +144,11 @@ def __eq__(self, other):
 else:
     field = [field]
 
+# many advisors in biology are chemists. 
```

Comment:
we can delete lines 147-151 because we work on this in #11

Reply:
done